ast/src/parsed/mod.rs

```diff
@@ -850,6 +822,47 @@ impl Children<Expression> for ArrayExpression {
     }
 }
 
+#[derive(Debug, PartialEq, Eq, PartialOrd, Ord, Clone, Serialize, Deserialize, JsonSchema)]
+pub enum Pattern {
+    CatchAll,
+    #[schemars(skip)]
```

Comment:
why skip?

Reply:
Because it's not implemented for the bigint type.

Reply:
What would that achieve? In Type we also don't group by elementary and non-elementary.

Reply:
idk, just feels too specific, but I guess it's fine since those are the only literals